includes/admin/class-wp-job-manager-promoted-jobs.php

```diff
@@ -54,6 +56,9 @@ public function __construct() {
 	 * @return array
 	 */
 	public function promoted_jobs_columns( $columns ) {
+		if ( ! $this->promote_jobs_template_exists() ) {
```

Comment:
The idea is nice, but I think we could keep this even in case of error because it could be something specific to the template.

If we implement something like this, I think it could be in some status endpoint in wpjobmanager.com, and turn off the whole Promoted Jobs feature.

WDYT?

Reply:
Removed the check for now in 52593c0b3bf8d2bf0eb843fc1cd5e7ae188be583

Are you thinking something like... if response is not 200, we check a status endpoint on wpjobmanager.com?  And if the status check fails, we turn off promoted jobs entirely?   Would that run every time we try to get the template?

Reply:
Related to this, we should also have a filter to turn off the whole promoted jobs feature, in case a dev wants to turn it off for a client site.

Reply:
We did discuss having a single point of entry so I'll also suggest moving this file to the `promoted-jobs` folder and including it from [here](https://github.com/Automattic/WP-Job-Manager/blob/594b31da03443d96a1f6527e4d1f55934aa6b90a/includes/promoted-jobs/class-wp-job-manager-promoted-jobs.php#L62-L65) maybe using `is_admin()` or alternatively using the `admin_init` hook.

Reply:
Once this is merged I'll do another PR to move the whole file over to the `promoted-jobs` folder and instantiate it properly.